lntest/btcd.go

```diff
@@ -17,11 +17,11 @@ import (
 // logDir is the name of the temporary log directory.
 const logDir = "./.backendlogs"
 
-// perm is used to signal we want to establish a permanent connection using the
+// temp is used to signal we want to establish a temporary connection using the
```

Comment:
can you add a more detailed explanation for the change to the commit message?

Reply:
sure, added.